dotnet-interactive/JupyterFormattingKernelExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.CSharp;
+using Microsoft.DotNet.Interactive.FSharp;
+using Microsoft.DotNet.Interactive.Jupyter.Formatting;
+
+namespace Microsoft.DotNet.Interactive.App
+{
+    public static class KernelFormattingExtensions
+    {
+        public static IKernel UseJupyterFormatting(this CSharpKernel kernel)
+        {
+            if (kernel == null)
+            {
+                throw new ArgumentNullException(nameof(kernel));
+            }
+
+            Task.Run(() =>
+                kernel.SendAsync(
+                    new SubmitCode($@"
+#r ""{typeof(LaTeX).Assembly.Location.Replace("\\", "/")}""
+using {typeof(LaTeX).Namespace};
+"))).Wait();
+
+            return kernel;
+        }
+
+        public static IKernel UseJupyterFormatting(this FSharpKernel kernel)
+        {
+            if (kernel == null)
+            {
```

Comment:
The return type should be `FSharpKernel`.

Reply:
done